src/Components/AddModal/AddModal.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { ChangeEvent } from 'react';
+import Select, { components } from 'react-select';
+import css from './AddModal.less';
+
+export function AddModal({ onReset, onSubmit }: { onReset: () => void, onSubmit: () => void }) {
+  const genres = [
+    { label: 'Documentary', value: 'documentary' },
+    { label: 'Comedy', value: 'comedy' },
+    { label: 'Horror', value: 'horror' },
+    { label: 'Crime', value: 'crime' },
+  ];
+
+  const dropdownIndicator = (props) => {
+    return (
+      <components.DropdownIndicator {...props}>
+        <span className={css.selectIcon} />
+      </components.DropdownIndicator>
+    );
+  };
+
+  function onDateChange(event: ChangeEvent<HTMLInputElement>) {
+    if (event.target.value) {
+      event.target.classList.add(css.full);
+    } else {
+      event.target.classList.remove(css.full);
+    }
+  }
+
+  return (
+    <>
+      <h2>ADD MOVIE</h2>
+      <form>
+        <fieldset>
+          <label htmlFor="title">TITLE</label>
+          <input type="text" name="title" id="title" placeholder="Movie title" />
+        </fieldset>
+        <fieldset>
+          <label htmlFor="date">RELEASE DATE</label>
+          <input
+            name="date"
+            id="date"
+            placeholder="Select release date"
+            type="date"
+            onChange={onDateChange}
+          />
+        </fieldset>
+        <fieldset>
+          <label htmlFor="url">MOVIE URL</label>
+          <input type="text" name="url" id="url" placeholder="Movie url" />
+        </fieldset>
+        <fieldset>
+          <label htmlFor="genre">GENRE</label>
+          <Select
+            isMulti={true}
+            options={genres}
+            classNamePrefix="react-select"
+            components={{ DropdownIndicator: dropdownIndicator }}
+          />
+        </fieldset>
+        <fieldset>
+          <label htmlFor="overview">OVERVIEW</label>
+          <input type="text" name="overview" id="overview" placeholder="Movie overview" />
+        </fieldset>
+        <fieldset>
+          <label htmlFor="runtime">RUNTIME</label>
+          <input type="text" name="runtime" id="runtime" placeholder="Movie runtime" />
+        </fieldset>
+        <div className={css.buttons}>
+          <button onClick={onReset}>RESET</button>
+          <button data-ui="primary" onClick={onSubmit}>SUBMIT</button>
```

Comment:
It's better to use onSubmit event on form instead of button clicking to allow users submitting form using a keyboard

Reply:
fixed